server/channels/app/imaging/preview.go

```diff
@@ -34,17 +34,13 @@ func GenerateThumbnail(img image.Image, width, height int) image.Image {
 	if h > height || w > width {
```

Comment:
Now that we simplified, would it make sense to rename `height` to `targetHeight` and `width` to `targetWidth`? A bit more verbose but it helps differentiating between inputs and outputs.

Reply:
Renamed.